content/ops/deploying-the-admin-ui.md

```diff
@@ -80,4 +64,4 @@ Or...
 
 #### Operation
 
-Access the Admin UI at the uri specified in the deployment manifest. By default this is `https://admin.YOUR-SYSTEM-DOMAIN` as generated by `meta.subdomain "." properties.system_domain` in the `templates/admin-ui-deployment.yml`.
+Access the Admin UI at the uri specified in the deployment manifest. By default this is `https://admin.YOUR-SYSTEM-DOMAIN` as generated by `meta.subdomain "." properties.system_domain` in the `templates/admin-ui-deployment.yml`.  If you did not change this setting, try `https://admin.bosh-lite.com`.
```

Comment:
It was a mistake before - there was something which caused my DNS not to resolve bosh-lite.com, causing the Vagrant host not to work.  After a full repeat of the initial instructions, I found it was no longer needed.  May have to do with VPN but I haven't reproduced it.